index.html

```diff
@@ -0,0 +1,66 @@
+!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ]  You should fix the typo error so that your design matches with the one from figma. 
![image](https://user-images.githubusercontent.com/43877689/177412344-239d8271-563c-4688-8cb5-1325c51f39f5.png)